context/app/static/js/hooks/useEntityData.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
I think this does the same thing as the [LookupEntity](https://github.com/hubmapconsortium/portal-ui/pull/610/files#diff-606a7a99a80c265ec783d37535884a59) I introduced over the weekend... Let's just keep one? No strong preference between the two.

Reply:
I'd prefer to use the hook.